jetty-core/jetty-http/src/test/java/org/eclipse/jetty/http/HttpParserTest.java

```diff
@@ -557,12 +557,23 @@ public void testNoValue(String eoln)
         ByteBuffer buffer = BufferUtil.toBuffer(
             "GET / HTTP/1.0" + eoln +
                 "Host: localhost" + eoln +
-                "Name0: " + eoln +
+                "Name0:  " + eoln +
                 "Name1:" + eoln +
+                "Authorization:  " + eoln +
+                "Authorization:" + eoln +
                 eoln);
 
-        HttpParser.RequestHandler handler = new Handler();
+        HttpParser.RequestHandler handler = new Handler()
+        {
+            @Override
+            public void badMessage(HttpException failure)
```

Comment:
Is this override still needed?

Reply:
I added it as before the fix it was very unclear how it was failing.